backend/logic/discount/index.js

```diff
@@ -199,8 +196,8 @@ const validateDiscountOnOrder = async (orderObj) => {
       }
     }
 
-    if (maxDiscountValue) {
-      discount = Math.min(maxDiscountValue, discount)
+    if (maxDiscountValue && ['percentage', 'payment'].includes(discountType)) {
```

Comment:
How does maxDiscountValues get set for an Automatic discount? I don't see that option being available in the admin UI? I assume it's for the future (which is a good thing to have)?

Reply:
Yeah, at present, you can set the `maxDiscountValue` for only 'percentage' discountType. The UI doesn't have a field for this in automatic discounts screen.